assets/src/application/ui/layout/entityActionsMenu/entityMenuItems/Trash.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+
+import { EspressoDropdownMenuItem } from '@application/ui/layout/espressoDropdownMenu';
+import { Icon } from '@application/ui/input';
+import { MenuItemProps } from './types';
+
+const Trash: React.FC<MenuItemProps> = ({ entityType, onClick, ...props }) => {
+	const title = sprintf(__('trash %s'), entityType);
```

Comment:
`entityType` is [not being translated](https://github.com/eventespresso/event-espresso-core/pull/2546/files#diff-bd9d38bb96c35744dcd2864ab0732210R31) from the source. It's better to explicitly get the title from consumer and set default to `__('copy')`. The consumer can then pass `__('copy datetime')`

Reply:
updated, thx:
https://github.com/eventespresso/event-espresso-core/pull/2546/commits/62b59b24f731f99dcf120e00d95aeea6ce616af0